factory-containers/assemble-system-image

```diff
@@ -181,7 +184,11 @@ if __name__ == '__main__':
     wic_tool = args.wic_tool
 
     try:
-        targets = get_targets_from_api(args.factory, args.build_num, args.token)
+        if args.targets:
+            targets = json.loads(args.targets)
```

Comment:
so this is a list of full blown Tuf Targets, right? ie - its not just a list target keys? Is it possible to share a little more context on how this is going to get called?

Reply:
>so this is a list of full blown Tuf Targets, right?
yes

The JobServe CI job that runs this script will be triggered by the ota-lite service, the ota-lite part will be triggered by fioctl as we discussed. The ota-lite API/handler will be HTTP POST targets/<target>/images.